crates/sc-proof-of-time/src/time_keeper.rs

```diff
@@ -98,17 +98,19 @@ where
 
     /// Initializes the chain state from the consensus tip info.
     async fn initialize(&self) {
-        info!("time_keeper::initialize: waiting for initialization ...");
+        debug!("Waiting for initialization");
```

Comment:
Why was this changed? Without target and no prefix, the logs don't have much context

Reply:
They actually do, as mentioned in previous PR reviews, these are excessive and hard to read, logger already prints the module when you log something and you don't need to do it manually, here is an example:
```
2023-08-17T18:22:46.374851Z DEBUG single_disk_farm{disk_farm_index=0}: subspace_farmer::single_disk_farm::farming: Solution found slot=846148283 sector_index=9
2023-08-17T18:22:46.379039Z  INFO single_disk_farm{disk_farm_index=0}: subspace_farmer::reward_signing: Successfully signed reward hash 0x1f4767f4785b35cf0766f783e076356a3f0794aed2cfc10ad8647396aad173d0
```

And while I understand you're still developing it, `info` level should be used very occasionally for things users would actually care about and understand, hence downgrade to `debug`.